backend/tracim_backend/models/data.py

```diff
@@ -1395,6 +1395,21 @@ def recursive_children(self) -> List["Content"]:
             )
         return []
 
+    @property
```

Comment:
Why not directly constructing the content path here?

Reply:
somes reason:
- Where at the the level of the content itself, for me the concept of "path" doesn't make sense at this level.
- We do already have recursive_children, it make sense to have also recursive_parent.
- recursive parent may be useful for some user case.
- If we decide later that the path of a content is "workspace path + content path", it will be way more easier to do the work at the Lib level than on content level.

Reply:
I think `path` does make sense at this level: when you got a file in the filesystem it is usual to be able to get its path directly I think.
I we want to keep the data classes as only "data" we could move this method to `ContentApi`.

Reply:
i suggest that we move the content_path here (which will be a content based path (not workspace) but we do keep recursive_parents method (content_path will use recursive_parents content).
There 2 differences : 
- path contain content itself
- reversed order.